cypress/support/step_definitions/dashboard/listitems/i_should_see_{string}_with_a_value_of_{string}_on_row_number_{string}.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable */
+Then(
+  "I should see {string} with a value of {string} on row number {string}",
+  (rowLabel, rowValue, rowPosition) => {
+    const rowAtPosition = (position) =>
+      cy
+        .get(".govuk-summary-list > .govuk-summary-list__row")
+        .eq(Number(position) - 1);
+
+    rowAtPosition(rowPosition).contains("dt", rowLabel);
+    rowAtPosition(rowPosition).contains("dd", rowValue);
```

Comment:
https://testing-library.com/docs/queries/about#priority 

I would follow this first, before resorting to css selectors. By using selectors based on roles, you can be more confident that the element you're trying to access is accessible to a screenreader. In general, css classes are a recipe for flakiness when it comes to testing